packages/import-bundle/src/index.js

```diff
@@ -124,6 +174,21 @@ export async function importBundle(bundle, options = {}, powers = {}) {
   }
 }
 
+/**
+ * A utility function for producing test bundles, which are not seerializable
+ * as JSON or passable.
+ * @param {Record<PropertyKey, unknown>} exports
+ */
+export const bundleTestExports = exports => {
+  const symbols = Object.getOwnPropertySymbols(exports);
+  symbols.length > 0 &&
+    Fail`exports must not have symbol-named properties, got: ${symbols.map(String).join(', ')}`;
+  return {
+    moduleFormat: 'test',
+    [Symbol.for('exports')]: exports,
+  };
```

Comment:
Should we freeze the object (not harden it since we don't care about the stability of exports at this point)

Reply:
That would be a departure from the norm for `bundleSource`. I think it’s reasonable to defer that choice to the caller. But, if we at some point reörganize the bundler infrastructure and create a new entry-point, I’d be open to changing the contract. Other reasons to consider reorganizing around an `@endo/bundler` module maybe someday would be to consolidate `@endo/import-bundle`, `@endo/bundle-source`, and `@endo/check-bundle` into a single package so they can more easily cross-test, do a better job of isolating powers and decoupling them from the underlying platform (like `node-powers.js` in the Compartment Mapper), remove deprecated powers like the ones taken by Rollup, one README for all the bundle formats and how to read and write them, &c. The list is growing but not pressing.